tests/cli.rs

```diff
@@ -144,6 +144,25 @@ fn list_mutants_in_factorial_json() {
         .assert_insta("list_mutants_in_factorial_json");
 }
 
+#[test]
+fn list_mutants_in_meta_tree() {
+    run()
+        .arg("mutants")
+        .arg("--list")
+        .current_dir("testdata/tree/meta_tree")
+        .assert_insta("list_mutants_in_meta_tree");
+}
+
+#[test]
+fn list_mutants_in_meta_tree_json() {
+    run()
+        .arg("mutants")
+        .arg("--list")
+        .arg("--json")
+        .current_dir("testdata/tree/meta_tree")
+        .assert_insta("list_mutants_in_meta_tree_json");
+}
+
```

Comment:
I think the test failure in this tree probably indicates we should also run `cargo mutants` on a copy of this source tree.